Release notes hotfix part 2 (#3418)
astronomer/docs: astro/cli/release-notes.md

    - If you log in to Astro with an API token using the `ASTRO_API_TOKEN`, `ASTRONOMER_KEY_ID`, or `ASTRONOMER_KEY_SECRET` environment variables, auto-select is enabled. This is important because it ensures that if you have CI/CD scripts that rely on auto-select, they will continue to work.
    - There is a new config, `auto_select`. If `auto-select` is set to `true` in the config file, auto-select is always enabled.

### Bug fixes

> - Fixed a bug where `astro dev pytest --args` and `astro dev pytest --build-secrets` would fail.

Reviewer: ```suggestion

- Fixed an issue where `astro dev pytest --args` and `astro dev pytest --build-secrets` could fail.
```
